src/audio/Kconfig

```diff
@@ -142,6 +142,15 @@ config COMP_DRC
 	  to reduce the volume of loud sounds and amplify silent sounds thus
 	  compressing an audio signal's dynamic range.
 
+config COMP_MULTIBAND_DRC
+	depends on COMP_CROSSOVER && COMP_DRC
```

Comment:
In fact MULTIBAND_DRC is not dependent on IIR, because IIR calculation (iir_df2t) is defined in src/math now rather than src/audio/eq_iir and it is the only function MULTIBAND_DRC refers.
https://github.com/thesofproject/sof/blob/master/src/math/iir_df2t_generic.c#L47

Reply:
I will be undoing that though here soon since the math lib repurposed because of the weird dyn lib design of the testbench, i think we should add it anyways.

Reply:
Added